scene/gui/control.cpp

```diff
@@ -2347,8 +2347,7 @@ Control *Control::_get_focus_neighbor(Side p_side, int p_count) {
 	};
 
 	Vector2 vdir = dir[p_side];
-
-	Rect2 r = get_global_rect();
+	Rect2 r = get_screen_transform().xform(Rect2(Point2(), get_size()));
```

Comment:
Not sure if screen transform is correct here. It might break inside SubViewports.
Normally you'd use global transform for that.

Reply:
1. The actions corresponding to `ui_*` should be relative to the screen.
2. The global transformation is relative to the canvas.

And because `CanvasLayer` can apply `rotation`. So the directions of the global coordinate system and the screen coordinate system may not be consistent.